core/templates/services/interaction-specs.service.spec.ts

```diff
@@ -29,6 +29,14 @@ describe('InteractionSpecsService', () => {
 
   describe('checking whether an interaction can be trained with ML', () => {
     it('should throw an error when interaction does not exist.', () => {
+      // This throws "Argument of type '"Fake"' is not assignable to parameter
+      // of type '"ImageClickInput" | "NumberWithUnits" |
+      // "NumericExpressionInput" | "NumericInput" | "DragAndDropSortInput" |
+      // "ItemSelectionInput" | "Continue" | "GraphInput" | "EndExploration" |
+      // ... 11 more ... | "RatioExpressionInput"'." We need to suppress this
```

Comment:
Don't use `11` as the number of interactions can be variable, (`...` is enough!)